client/src/components/NavBar/NavBar.tsx

```diff
@@ -10,7 +10,7 @@ import {
 import { NavBkgdType } from '../../types';
 
 function NavBar({ navBackground }: NavBkgdType) {
-  const userInfo = useUserInfo();
+  const { userInfo } = useUserInfo();
```

Comment:
I'd recommend destructuring `userId` here to clean up the code below a bit:
```suggestion
  const { userInfo: { userId }} = useUserInfo();
```